generate-clang-meson-cross-file.sh

```diff
@@ -32,7 +34,7 @@ toolchain_cflags=("${@:3}")
 # In particular, the clang driver requires a `--gcc-toolchain=<path>` argument
 # to find the right libraries if there are system versions of the risc-v
 # toolchains installed.
-meson_driver_flags="'--gcc-toolchain=${toolchain_dest}'"
+meson_driver_flags="toolchain_opt"
```

Comment:
Should this be without the quotes, maybe?

Reply:
I guess I was following the existing style in this repo, which seems to use quotes even when it's not really needed. I don't have strong feelings about this either way.

Reply:
Ohh! I'd misunderstood and thought that we were evaluating a `toolchain_opt` variable at this point. But we're not! We're just passing its name along for Meson to make sense of later. Looks good to me.